libraries/AP_VideoTX/AP_VideoTX.h

```diff
@@ -204,6 +209,9 @@ class AP_VideoTX {
     AP_Int8 _enabled;
     bool _current_enabled;
 
+    // vtx temperature, not all backends provide it
+    uint16_t _current_temperature;
```

Comment:
You need a way of showing this is unavailable (-1) and use it to disable the OSD element

Reply:
introduced -1.0f as invalid, it will be handled in the next step to show something like '----' .